style.css

```diff
@@ -1 +1,160 @@
 /*  Add your styling here */
+
+/* hamburger */
+
+/* head */
+header {
+  display: flex;
+  flex-direction: column;
+  margin: 0;
+}
+.image-container {
+  display: flex;
+  flex-direction: row;
+  background-color: rgb(254, 254, 254);
+}
+.logo {
+  flex: 20%;
+  margin: 0;
+}
+.logo img {
+  width: 50%;
+}
+.textMenu-container {
+  flex: 60%;
+  text-align: right;
+}
+.textMenu-container p {
+  flex: 60%;
+  text-align: right;
+  color: rgb(85, 5, 5);
+}
+.textMenu-container img {
+  float: right;
+}
+.text {
+  margin-top: 50px;
+  margin-right: 50px;
+  font-size: 20px;
+}
+.hamburger-menu {
+  width: 100%;
+  display: flex;
+  justify-content: space-around;
+  background-color: rgb(179, 176, 176);
+}
+
+.hamburger-menu a {
+  display: none;
+}
+.hamburger-item {
+  text-decoration: none;
+  color: rgb(85, 5, 5);
+  font-size: 20px;
+}
+.hamburger-menu a {
+  text-decoration: none;
+}
+
+/* main*/
+#intro-container {
+  width: 100%;
+  text-align: justify;
+  display: block;
+  background-color: rgb(230, 230, 230);
+  color: rgb(128, 8, 8);
+}
+
+.intro-image {
+  width: 100%;
+  margin: auto;
+  margin-bottom: 20px;
+}
+.intro-heading {
+  text-align: left;
+}
+
+.intro-text {
+  text-align: left;
+}
+
+/*product*/
+#product-container {
+  width: 100%;
+  display: flex;
+}
+
+.product-text {
+  text-align: center;
+  color: rgb(179, 146, 40);
+}
+
+.product-image {
+  width: 45%;
+  justify-content: space-between;
+  padding: 5px;
+}
+
+/* Footer*/
+#Footer-container {
+  width: 100%;
+  display: flex;
+  justify-content: space-around;
+  background-color: rgb(179, 176, 176);
+}
+
+.footer-item {
+  text-decoration: none;
+  color: rgb(255, 226, 231);
+  font-size: 10px;
+}
+
+@media (min-width: 540px) {
+  /* head*/
+  .hamburger-menu a {
+    display: flex;
+    font-size: 25px;
+  }
+  .menu-icon {
+    display: none;
+  }
+
+  /* main*/
+  #intro-container {
+    display: flex;
+    flex-direction: row-reverse;
+  }
+
+  .intro-image {
+    justify-content: left;
+    width: 90%;
+  }
+
+  .intro-heading {
+    text-align: right;
+    padding-right: 40px;
+  }
+
+  .intro-text {
+    text-align: justify;
+    padding-right: 40px;
+  }
+
+  /*product*/
+
+  #Footer-container {
+    flex-direction: row;
+  }
+
+  .product-image {
+    width: 21%;
+    justify-content: space-between;
```

Comment:
Using flexbox rules without `display: flex` has no effect. This rule will not do anything until you make `product-image` a flex container.